src/plugins/rating/RatingHistoryEntry.tsx

```diff
@@ -38,16 +35,16 @@ const RatingHistoryEntry = styled.div(({ theme }) => ({
 }));
 
 export default ({ message, config }: MessageComponentProps) => {
-	const rating = message?.data?._cognigy?.controlCommands?.[0]?.parameters;
+	const rating = getRating(message);
 
 	const ratingText = config.settings.ratingMessageHistoryRatingText;
 	const commentText = config.settings.ratingMessageHistoryCommentText;
 
 	if (!rating) return null;
 
 	const getRatingIcon = (rating: number) => {
-		if (rating === 10) return <StyledThumbIcon />;
-		if (rating === 0) return <StyledThumbDownIcon />;
+		if (rating === 10) return <StyledThumbIcon style={{ marginLeft: 6 }} />;
```

Comment:
would be nicer to move inline styling to styled component (adding it once in line 11 should be enough)

Reply:
@pedily Will you be addressing this? Else, I will merge the PR.